debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/OracleValueConverters.java

```diff
@@ -151,6 +182,7 @@ public ValueConverter converter(Column column, Field fieldDefn) {
             case Types.NUMERIC:
                 return getNumericConverter(column, fieldDefn);
             case Types.FLOAT:
+                // todo: DBZ-3078 LogMiner used getFloatConverter here, review impact.
```

Comment:
Are you planning to address these todos as part of this PR, or as a follow-up?

Reply:
I am, they'll be fixed in the next commit I'm working on.